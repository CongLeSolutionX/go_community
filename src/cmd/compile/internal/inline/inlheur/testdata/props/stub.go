// Copyright 2023 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// DO NOT EDIT (use 'go test -v -update-expected' instead.)
// See cmd/compile/internal/inline/inlheur/testdata/props/README.txt
// for more information on the format of this file.
// =^=^=

package stub

var M = map[string]int{}

// stub.go T_stub 18
// =====
// {"Flags":0,"RecvrParamFlags":null,"ReturnFlags":null}
// =-=-=
func T_stub() {
}

func ThisFunctionShouldBeIgnored(x int) {
	println(x)
}

// stub.go init.0 29
// =====
// {"Flags":0,"RecvrParamFlags":null,"ReturnFlags":null}
// =-=-=
func init() {
	ThisFunctionShouldBeIgnored(1)
}
