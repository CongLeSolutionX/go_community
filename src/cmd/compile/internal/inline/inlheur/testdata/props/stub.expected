// DO NOT EDIT (use 'go test -update-expected' instead))
TStub
===
{
	"Flags": 0,
	"RecvrParamFlags": null,
	"ReturnFlags": null
}
=-=-=-
